src/components/form/Form.tsx

```diff
@@ -1,161 +1,94 @@
 import LabelField from "../inputAndLabel/LabelField.tsx";
-import InputField from "../inputAndLabel/InputField.tsx";
-
-import { Color } from "../colors.ts";
-import {
-  ChangeEvent,
-  KeyboardEvent,
-  SyntheticEvent,
-  useEffect,
-  useState,
-} from "react";
-
+import { useForm } from "react-hook-form";
 import FormButton from "./FormButton.tsx";
-import { nanoid } from "nanoid";
-
+import { Color } from "../colors.ts";
 import { StyledForm } from "./Form.styles.ts";
-
-//retrieve data from local storage
-//if no data is saved return empty object
-const getPlants = () => {
-  const storedPlants = JSON.parse(localStorage.getItem("formData"));
-  if (!storedPlants)
-    return {
-      plants: [
-        {
-          plantID: "",
-          plantName: "",
-          plantImage: "",
-        },
-      ],
-    };
-  return storedPlants;
-};
+import { nanoid } from "nanoid";
+import InputField from "../inputAndLabel/InputField.tsx";
+import NavItem from "../navItem/NavItem.tsx";
 
 const Form = () => {
-  const [initialFormData, setInitialFormData] = useState(getPlants);
-  const [singlePlant, setSinglePlant] = useState(initialFormData.plants[0]);
-
-  //IMAGE PROBABLY WILL BE ADDED IN ANOTHER ITERATION
-  // const [image, setImage] = useState('')
-  //
-  // const onImageChange = (event) => {
-  //     setImage(URL.createObjectURL(event.target.files[0]))
-  // }
+  const storedPlants = JSON.parse(localStorage.getItem("formData"));
 
-  //control inputs and create ID for single plant
-  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
-    const { name, value }: EventTarget & HTMLInputElement = event.target;
-    const newPlantID = nanoid();
+  const plantsArray = storedPlants ? storedPlants : [];
```

Comment:
refactor with `||`

Reply:
and generally you are cloning `storedPlants` for no apparent reason :) maybe a fallback to the line above would be enough?

Reply:
and names including words like `array` or `object` because those are reserved words. You can use e.g. `collection` instead or some other synonym.

Reply:
fixed!